qrcode/image/base.py

```diff
@@ -15,8 +17,8 @@ class BaseImage(abc.ABC):
     Base QRCode image output class.
     """
 
-    kind: Optional[str] = None
-    allowed_kinds: Optional[tuple[str]] = None
+    kind: str | None = None
+    allowed_kinds: tuple[str] | None = None
```

Comment:
This type doesn't seem correct, by the way – `tuple[str]` is a 1-tuple of a single string, `tuple[str, ...]` is a homogeneous N-tuple.
```suggestion
    allowed_kinds: tuple[str, ...] | None = None
```